src/components/Chat/MessageWithImage.js

```diff
@@ -176,7 +176,7 @@ const styles = StyleSheet.create({
   textMore: {
     fontSize: 24,
     fontFamily: fonts.inter[400],
-    color: COLORS.white
+    color: COLORS.almostBlack
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the text has been changed from white to almost black. This change should be reviewed in the context of the background color to ensure that there is enough contrast for readability.
```diff
-    color: COLORS.white
+    color: COLORS.almostBlack
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->